src/app/layouts/header/export-popup/export-popup.component.ts

```diff
@@ -73,6 +76,34 @@ export class ExportPopupComponent implements OnInit, OnDestroy {
         this.popup.hide();
     }
 
+    exportZip(): void {
+        const zip = new JSZip();
+        const generic = ['content/css/icons/dxicons.ttf', 'content/css/icons/dxicons.woff', 'content/css/icons/dxicons.woff2'];
```

Comment:
```suggestion
        const genericFiles = ['content/css/icons/dxicons.ttf', 'content/css/icons/dxicons.woff', 'content/css/icons/dxicons.woff2'];
```

Reply:
Nah, let's have an array of file extensions and `dxicons` or `dxiconsmaterial` in the `choice` variable. It's a bit shorter.